Examples/IO/XML/itkParticleSwarmOptimizerSAXReader.cxx

```diff
@@ -27,6 +27,7 @@ namespace itk
 /**
  * Check that whether the file with given name is readable.
  */
+/** @ITKStartGrouping */
```

Comment:
Do we need this in examples?

Reply:
No not really but they automatically appeared as they did with the script. They don't hurt though.
It would be probably good to iron them out, but I first made the "brute force approach" so the warnings are gone and we have a better code base.